centos7/Dockerfile.crunchy-postgres-exporter.centos7

```diff
@@ -0,0 +1,42 @@
+ARG BASEVER
+ARG PREFIX
+ARG PGVERSION
+FROM ${PREFIX}/pgo-base:centos7-${BASEVER}
+
+LABEL name="crunchy-postgres-exporter" \
+	summary="Metrics exporter for PostgreSQL" \
+	description="When run with the crunchy-postgres family of containers, crunchy-postgres-exporter reads the PostgreSQL data directory and has a SQL interface to a database to allow for metrics collection." \
+	io.k8s.description="Crunchy PostgreSQL Monitoring with pgMonitor - Collector" \
+	io.k8s.display-name="Crunchy PostgreSQL Monitoring with pgMonitor - Collector" \
+	io.openshift.tags="postgresql,postgres,monitoring,database,crunchy"
+
+RUN yum -y install epel-release \
+    && yum install -y \
+        --setopt=skip_missing_names_on_install=False \
+        postgresql${PGVERSION} \
+    && yum clean all -y
+
+RUN mkdir -p /opt/cpm/bin /opt/cpm/conf
+
+ADD postgres_exporter.tar.gz /opt/cpm/bin
+ADD tools/pgmonitor/exporter/postgres /opt/cpm/conf
+ADD bin/crunchy-exporter /opt/cpm/bin
+ADD conf/crunchy-exporter /opt/cpm/conf
+
+RUN chgrp -R 0 /opt/cpm/bin /opt/cpm/conf && \
+	chmod -R g=u /opt/cpm/bin/ opt/cpm/conf
+
+# postgres_exporter
+EXPOSE 9187
+
+RUN chmod g=u /etc/passwd
+
+# The VOLUME directive must appear after all RUN directives to ensure the proper
+# volume permissions are applied when building the image
+VOLUME ["/conf"]
+
+ENTRYPOINT ["/opt/cpm/bin/exporter_uid_daemon.sh"]
+
+USER 26
```

Comment:
I see that that the entrypoint calls a script to set the UID for the daemon user, but the user has been updated to `26` here (i.e. the postgres user).  Should this be `USER 2`?

Reply:
Yes, in the current design, that should now be 2. Updated.